specs/deneb/p2p-interface.md

```diff
@@ -117,6 +117,11 @@ Deneb introduces new global topics for blob sidecars.
 
 The *type* of the payload of this topic changes to the (modified) `SignedBeaconBlock` found in deneb.
 
+New validation:
+
+- _[REJECT]_ The length of KZG commitments is less than or equal to the limitation defined in Consensus Layer --
+  i.e. validate that `len(body.signed_beacon_block.message.blob_kzg_commitments) <= MAX_BLOBS_PER_BLOCK`
```

Comment:
@djrtwo I added the p2p validation here. Feel free to open a post-merge PR to revise it.

Reply:
yep, makes sense!. definitely need the p2p limit given the type isn't protecting us from dos